README.md

```diff
@@ -132,7 +132,7 @@ Bamboo offers `deliver_later` on your mailers to send emails in the background s
 By default delivering later uses [`Bamboo.TaskSupervisorStrategy`](https://hexdocs.pm/bamboo/Bamboo.TaskSupervisorStrategy.html). This strategy sends the email right away, but does so in the background without linking to the calling process, so errors in the mailer won't bring down your app.
 
 If you need something more custom you can
-create a strategy with [Bamboo.DeliverLaterStrategy](https://hex.pm/packages/bamboo). For example, you could create strategies
+create a strategy with [Bamboo.DeliverLaterStrategy][]. For example, you could create strategies
```

Comment:
This cane like the link on line `132` where it just has the `[]` inside of backpacks. Then hex docs will automatically find the module and link to it :D